Gemfile

```diff
@@ -71,6 +75,7 @@ group :test do
   gem 'selenium-webdriver'
   # Easy installation and use of chromedriver to run system tests with Chrome
   gem 'chromedriver-helper'
+  gem 'vcr', '~> 4.0'
```

Comment:
doesn't look like you're making use of this gem. would be worth getting it configured and wrapped around the new location specs

Reply:
Configured